trackpy/utils.py

```diff
@@ -16,6 +17,14 @@
 
 import trackpy
 
+# Set is_pandas_recent for use elsewhere.
+# Pandas >= 0.16.0 lets us check if a DataFrame is a view.
+try:
+    is_pandas_recent = (StrictVersion(pd.__version__) >=
+                        StrictVersion('0.16.0'))
+except ValueError:  # Probably a release candidate
```

Comment:
From the "rc" in the version string. We could use `LooseVersion`, which allows alphabetic characters. But I figured that the test needed a fallback anyhow, so I went for that.